content/docs/guides/vercel.md

```diff
@@ -131,34 +130,51 @@ If you added the Neon Vercel Integration to a single Vercel project but would li
    1. Click **Save**.
 1. Navigate to this URL: [https://vercel.com/integrations/neon/new](https://vercel.com/integrations/neon/new).
-1. Follow the prompts. When you reach the **Integrate Neon** dialog, select the Vercel project that you want to add the integration to. Vercel projects that are already integrated with Neon are identified as `CONNECTED`.
+1. Follow the prompts. When you reach the **Integrate Neon** dialog, select the Vercel project that you want to add the integration to. Vercel projects that are already integrated with Neon are identified as `CONNECTED`, so you'll have to select a different Vercel project.
    ![Confirm integration settings](/docs/guides/vercel_add_new_project.png)
-1. Complete the steps outlined in [Add the Neon integration](#add-the-neon-vercel-integration).
+1. Continue following the prompts to complete the setup. These are the same steps described above, in [Add the Neon integration](#add-the-neon-vercel-integration). When you select a Neon project to connect to, be sure to select one that is not already connected to a Vercel project, as you cannot connect one Vercel project to multiple Neon projects or vice versa. 
 
 ## Troubleshoot connection issues
 
+This section describes commonly encountered connection issues for the Neon Vercel Integration.
+
+### Failed to set environment variables
+
 If the environment variables configured by the Neon integration already exist, you may encounter the following error due to an existing integration that sets one or more of the same environment variables.
 
+<CodeBlock shouldWrap>
+
 ```text
 Failed to set environment variables in Vercel. Please make sure that the following environment variables are not set: PGHOST, PGUSER, PGDATABASE, PGPASSWORD, DATABASE_URL
 ```
 
+</CodeBlock>
+
 In this case, you can remove or rename the existing environment variables in your Vercel project settings and retry the Neon integration.
 
 1. From the Vercel project page, select **Settings**.
-1. Locate the environment variables required by the Neon integration and remove or rename them.
+2. Locate the environment variables required by the Neon integration and remove or rename them.
 
-<Admonition type="note">
-Alternatively, you can remove the conflicting integration, assuming it no longer required. This may be a previous Neon integration or another integration. Removing the integration removes the variables set by the integration.
-</Admonition>
+   <Admonition type="note">
+   Alternatively, you can remove the conflicting integration, assuming it no longer required. This may be a previous Neon integration or another integration. Removing the integration removes the variables set by the integration.
```

Comment:
```suggestion
   Alternatively, you can remove the conflicting integration, assuming it is no longer required. This may be a previous Neon integration or another integration. Removing the integration removes the variables set by the integration.
```